activerecord/lib/active_record/connection_adapters/mysql/database_statements.rb

```diff
@@ -15,8 +15,8 @@ def select_all(arel, name = nil, binds = [], preparable: nil)
 
         # Returns an array of arrays containing the field values.
         # Order is the same as that returned by +columns+.
-        def select_rows(sql, name = nil, binds = [])
```

Comment:
We can remove this documentation and mark it as `nodoc` since it is the same documentation of the abstract.